### ISSUE TITLE
otu naming

### ISSUE THREAD

**Issue by antagomir** [microbiome/microbiome#77]
Thanks to XXX: " - - This solution does not resolve all duplicates in the strain level data, so in the profile-level phyloseq object taxa rownames are simply indexes ‘sp’ * This is an issue because some default plotting commands don’t provide as nicely labeled output as they could. (How do you interpret a figure where the names are codes like ‘sp203’?) * Figure out how to replace code-like names in the figures with proper names? - - "

-> One solution could be to have an option in the plotting command that would allow using alternative labels and/or more informative lables constructed from tax_table

**Comment by microsud** [comment]
Yes this would be good. Not yet sure how to do this but the idea here can be using the taxonomic classification like phylum:class: order:family:genus:species:otuid. Or simply the best taxonomic classification with more than 4 characters. Something similar to the solution we provide for core heat map issue. The number of levels to be included in labels can be specified by user.

**Comment by antagomir** [comment]
We now moved the development to mia pkg.
